internal/querycoordv2/balance/rowcount_based_balancer.go

```diff
@@ -184,7 +206,64 @@ outer:
 		node.setPriority(node.getPriority() + int(s.GetNumOfRows()))
 		queue.push(node)
 	}
-	return plans, nil
+	return plans, b.getChannelPlanForStoppingNodes(replica, stoppingNodes)
```

Comment:
Maybe we need to consider the priority of the balance task for stopping nodes, that should offline as soon as possible.